tests/ts/liblastlog2/dlopen

```diff
@@ -6,10 +6,21 @@ TS_DESC="dlopen"
 . "$TS_TOPDIR"/functions.sh
 ts_init "$*"
 
-echo -n ${top_builddir}/.libs/liblastlog2.so
+if [ -z "${top_builddir}" ]; then
+  # This should never happen
+  exit 1
+fi
+
+if [[ $top_builddir == *build ]]; then
+  #meseon build
+  libpath=${top_builddir}/liblastlog2
+else
+  #automake/autoconf build
+  libpath=${top_builddir}/.libs
+fi
```

Comment:
This also won't work, the build directories can be anywhere with any name.

In `tests/functions.sh` we test for `$top_builddir/meson.conf`. That is probably the easiest thing for now.